### DISCUSSION TITLE
Maybe need to support Markdown to show some variable or code block

### DISCUSSION THREAD

**Discussion by huanghui1998hhh** [h65wang/flutter_interview_questions#58] [Ideas]
Have there any other ideas, or some work around?

**Comment by JeffersonHuang** [comment]
Good idea.

**Comment by h65wang** [comment]
Agreed, although maybe not the full MD.

> Perhaps just \*italics* and \*\*bold** and \`code` should be enough?

Perhaps just *italics* and **bold** and `code` should be enough?

**Comment by JeffersonHuang** [comment]
Do we need markdown or do we need a CodeViewer Widget that only displays code.

**Reply by JeffersonHuang** [reply]
@huanghui1998hhh
